yarn.lock

```diff
@@ -4802,7 +4802,7 @@ create-hmac@^1.1.0, create-hmac@^1.1.4, create-hmac@^1.1.7:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
-cross-fetch@^3.0.5, cross-fetch@^3.0.6:
+cross-fetch@^3.0.5, cross-fetch@^3.0.6, cross-fetch@^3.1.5:
```

Comment:
Please run the `ncu` script once this is merged to resolve this problem with different packages being on different versions of things like `cross-fetch`.

Reply:
You're welcome to submit a PR :-)

Reply:
will do after this is merged to avoid conflicts